### DISCUSSION TITLE
iModelHub.UserDoesNotHavePermission

### DISCUSSION THREAD

**Discussion by harryhjsh** [iTwin/itwinjs-core#1130] [Q&A] [answered]
Hi,

I'm using the demo web viewer app with no changes (https://www.itwinjs.org/learning/tutorials/develop-web-viewer/).

I've made a sample project on the registration dashboard, and have pasted its context/model ID into the .env file.

When running npm start, I get this error in my browser once it's loaded: 
`Error: iModels.connectionError iModels.synchronizerLink`, 

and the console displays a 403 from `imodelhubapi.bentley.com` and `Error opening the iModel connection: iModelHub.UserDoesNotHavePermission` followed by what I assume is my user ID, then the correct model ID from the .env file.

Trying to look at the sample models on the Connect WebPortal, from the link on the registration dashboard also shows that I don't have Read access to these created sample models. Am I doing something wrong?

Thanks

**Comment by williamkbentley** [comment, marked as answer]
Hello,

We are having an RBAC issue with projects created through the registration dashboard. We are investigating and will reply back when we have an update.

Thanks

**Reply by harryhjsh** [reply]
Thought this might be the case - thanks for the rapid response

**Reply by williamkbentley** [reply]
I updated the role permissions. Can you give this another shot?

**Reply by harryhjsh** [reply]
Can confirm this is now working as expected with the sample app and viewing on the WebPortal

Thanks
